cmd/edv-rest/startcmd/start_test.go

```diff
@@ -236,18 +259,19 @@ func TestCreateProvider(t *testing.T) {
 		require.Equal(t, errInvalidDatabaseType, err)
 	})
 	t.Run("Error - CouchDB url is blank", func(t *testing.T) {
-		parameters := edvParameters{databaseType: databaseTypeCouchDBOption, databaseURL: ""}
+		parameters := edvParameters{databaseType: databaseTypeCouchDBOption, databaseURL: "", databaseTimeout: 1}
 
 		provider, err := createEDVProvider(&parameters)
 		require.Nil(t, provider)
-		require.Equal(t, couchdbedvprovider.ErrMissingDatabaseURL, err)
+		require.Equal(t, fmt.Errorf("fail to connect to couchdb: %w", couchdbedvprovider.ErrMissingDatabaseURL), err)
```

Comment:
Should be `failed` not `fail`

This applies to other places too

Reply:
fixed